guides/field-projection.md

```diff
@@ -0,0 +1,27 @@
+# Field Projection
+
+Some endpoints support field projection,
+which allows you to control which fields are returned in the response object.
+This is done with the use of fieldsets and projected fields.
+
+**Fieldsets** are predefined, named sets of fields.
+Fieldsets are provided by Wix for common use cases.
+For example, in the Contacts API,
+the `FULL` fieldset returns the full contact object,
```

Comment:
FMOK - do we have naming guidelines for what the fieldset names should be? Is it the standard to always use FULL and BASIC (in addition to others)? Or can dev choose anything they way, like ALL and BASIC,,for example? @imnotashrimp